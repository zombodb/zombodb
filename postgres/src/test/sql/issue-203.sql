select id, title from so_posts where zdb('so_posts', ctid) ==> '#limit(id asc, 0, 10) java and title:*' order by 1 asc;
select count(*) from so_posts where zdb('so_posts', ctid) ==> '#limit(id desc, 0, 10) beer and title:*';

select id from so_posts where zdb('so_posts', ctid) ==> '#limit(id asc, 0, 10) java and title:*' order by 1 asc;
select id from so_posts where zdb('so_posts', ctid) ==> '#limit(id asc, 10, 10) java and title:*' order by 1 asc;
