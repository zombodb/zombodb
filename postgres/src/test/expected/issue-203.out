select id, title from so_posts where zdb('so_posts', ctid) ==> '#limit(id asc, 0, 10) java and title:*' order by 1 asc;
  id  |                                   title                                    
------+----------------------------------------------------------------------------
   16 | Do dynamic typed languages deserve all the criticism?
  262 | Will Java still be relevant in 5 years?
  368 | Why do people still say Java is slow?
  370 | How old is "too old"?
  663 | What should be the ideal computer science and engineering course topics?
  667 | Writing a style guide, what are the best examples for breaking long lines?
  812 | Measure of power other than Turing completeness
 1189 | What should I do to be language-agnostic?
 1257 | How do you deal with teachers teaching the wrong stuff?
 1674 | Suggestions for Programs to develop for learning
(10 rows)

select count(*) from so_posts where zdb('so_posts', ctid) ==> '#limit(id desc, 0, 10) beer and title:*';
 count 
-------
    10
(1 row)

select id from so_posts where zdb('so_posts', ctid) ==> '#limit(id asc, 0, 10) java and title:*' order by 1 asc;
  id  
------
   16
  262
  368
  370
  663
  667
  812
 1189
 1257
 1674
(10 rows)

select id from so_posts where zdb('so_posts', ctid) ==> '#limit(id asc, 10, 10) java and title:*' order by 1 asc;
  id  
------
 1719
 1750
 3131
 3519
 3645
 4296
 4546
 4550
 4662
 4714
(10 rows)

